client/reader/following-manage/feed-search-results.jsx

```diff
@@ -44,17 +45,29 @@ const FollowingManageSearchFeedsResults = ( {
 		);
 	}
 
+	function recordResultRender( index ) {
```

Comment:
if you change this to:
`function recordResultRender( index, railcar ) --> () --> recordTracksRailcarRender ` then you can keep logic outside of the SubscriptionListItem.  it feels leaky to make something called `onRender` need to know about railcars.